src/libs/tagging.c

```diff
@@ -2667,20 +2667,31 @@ static gint _sort_tree_tag_func(GtkTreeModel *model, GtkTreeIter *a, GtkTreeIter
 {
   char *tag_a = NULL;
   char *tag_b = NULL;
+  char *tag_a_nc = NULL;
```

Comment:
let's move all this into an separate function (inlined) as the code in  `_sort_tree_path_func() ` is duplicate of this:

`static inline int _compare_utf8_no_case(const char *a, const char *b);`

Reply:
Good catch! I have now fixed this